pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -487,3 +515,12 @@ func handleResourceTracker(obj client.Object, limitingInterface workqueue.RateLi
 		}
 	}
 }
+
```

Comment:
Filter `ResourceVersion` in `func filterManagedFieldChangesUpdate`

Reply:
Fixed in #3008.